### ISSUE TITLE
copy new hmfs collections to cloud

### ISSUE THREAD

**Issue by JohnCremona** [LMFDB/lmfdb-inventory#80]
The HMFs code now on beta reads from collections forms/forms.rand/forms.stats making the collections forms.search (etc) obsolete.  Prior to pushing this code change to prod too I'll copy the three new collections to the cloud.  Eventually the forms.search* collections can be dropped (on beta they have already been renamed).

**Comment by JohnCremona** [comment]
I just did that and think it worked OK but there were some warnings since I used the update script on collections which were not already there:
```Renaming forms to forms.old and forms.new to forms in cloud database hmfs...
MongoDB shell version: 3.2.17
connecting to: hmfs
{ "ok" : 0, "errmsg" : "source namespace does not exist", "code" : 26 }
MongoDB shell version: 3.2.17
connecting to: hmfs
{ "ok" : 1 }
Renaming forms.stats to forms.stats.old and forms.stats.new to forms.stats in cloud database hmfs...
MongoDB shell version: 3.2.17
connecting to: hmfs
{ "ok" : 0, "errmsg" : "source namespace does not exist", "code" : 26 }
MongoDB shell version: 3.2.17
connecting to: hmfs
{ "ok" : 1 }
Renaming forms.rand to forms.rand.old and forms.rand.new to forms.rand in cloud database hmfs...
MongoDB shell version: 3.2.17
connecting to: hmfs
{ "ok" : 1 }
MongoDB shell version: 3.2.17
connecting to: hmfs
{ "ok" : 1 }
...update complete.
```

**Comment by JohnCremona** [comment]
This can be closed after #2357 on the main repository is closed. [That's the wrong number but I am not going to look for the right one, I am just closing this.]
